src/auto-updater/quit-and-install.js

```diff
@@ -10,9 +11,14 @@ module.exports = async function quitAndInstall ({ stopIpfs }) {
     browserWindow.removeAllListeners('close')
   })
 
-  const status = await stopIpfs()
+  try {
+    const status = await stopIpfs()
 
-  if (status === STATUS.STOPPING_FAILED || status === STATUS.STOPPING_FINISHED) {
+    if (status === STATUS.STOPPING_FAILED || status === STATUS.STOPPING_FINISHED) {
+      autoUpdater.quitAndInstall(true, true)
+    }
+  } catch (err) {
+    logger.error(err)
     autoUpdater.quitAndInstall(true, true)
   }
```

Comment:
I am probably too paranoid, but if `stopIpfs` returns anything other than [`6` or `2`](https://github.com/ipfs-shipyard/ipfs-desktop/blob/v0.11.3/src/daemon/consts.js) , nothing happens – no error, no update :grimacing: 

Is there any downside to just ignoring the status like this, and making `stopIpfs` a best-effort operation (that can't impact update mechanism in any way)?

```js
  try {
    await stopIpfs()
  } catch (err) {
    logger.error('unexpected error during stopIpfs', err)   
  }
  autoUpdater.quitAndInstall(true, true)
```

Reply:
Makes sense! Also added a logger info to help debug future problems with stopIpfs